src/zimscraperlib/video/encoding.py

```diff
@@ -32,11 +32,14 @@ def reencode(
 
     with tempfile.TemporaryDirectory() as tmp_dir:
         tmp_path = pathlib.Path(tmp_dir).joinpath(f"video.tmp{dst_path.suffix}")
-        args = (
-            ["ffmpeg", "-y", "-i", f"file:{src_path}"]  # noqa: RUF005
-            + ffmpeg_args
-            + [f"file:{tmp_path}"]
-        )
+        args = [
+            "ffmpeg",
```

Comment:
I wonder why the call using this is not complaining about a non absolute path

I also see we're not using `capture_output=True,` on the subprocess call…

Reply:
Its not the only one. I assume this is a ruff bug / limitation (it is only a code analysis tool, not a real type checker)

Should I fix all of them to use the `/usr/bin/env` trick?

Reply:
I think it'd be best to harmonize, yes

Reply:
Trick deployed.

We cannot use `capture_output=True` on the subprocess call, we are currently combining stderr and stdout, splitting them with `capture_output=True` would be a breaking change. Do you wanna open an issue for 4.x?

Reply:
Ah sorry